tests/integration_tests.rs

```diff
@@ -118,6 +111,9 @@ async fn test_wascc_provider() -> Result<(), Box<dyn std::error::Error>> {
 
     assert!(went_ready, "pod never went ready");
 
+    // Allow some time for the pod to actually become ready
+    tokio::time::delay_for(std::time::Duration::from_secs(2)).await;
```

Comment:
This is what the watcher loop is supposed to be doing. It is not actually ready when it hits running?